http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/KubernetesServiceDbQueries.scala

```diff
@@ -309,7 +330,7 @@ object KubernetesServiceDbQueries {
         case (appRec, (services, namespaces, labelMap, diskMap, errors)) =>
           appQuery.unmarshalApp(
             appRec,
-            services.map(serviceQuery.unmarshalService).toList,
+            services.map(serviceQuery.unmarshalService).toList.toSet.toList,
```

Comment:
`.toSet.toList`?

Reply:
Since for performance reasons its a chain prior to this, and Chain doesn't support toSet, we have to do it this way.